src/cdk-experimental/ui-patterns/tabs/tab.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import {computed, Signal} from '@angular/core';
+import {ListSelection, ListSelectionItem} from '../behaviors/list-selection/list-selection';
+import {ListNavigation, ListNavigationItem} from '../behaviors/list-navigation/list-navigation';
+import {ListFocus, ListFocusItem} from '../behaviors/list-focus/list-focus';
+import {TabpanelPattern} from './tabpanel';
+
+interface TablistPattern {
+  focusManager: ListFocus<TabPattern>;
+  selection: ListSelection<TabPattern>;
+  navigation: ListNavigation<TabPattern>;
+}
+
+/** The required inputs to tabs. */
+export interface TabInputs extends ListNavigationItem, ListSelectionItem, ListFocusItem {
+  tablist: Signal<TablistPattern>;
+  tabpanel: Signal<TabpanelPattern>;
```

Comment:
It seems odd to me that the tab would need to know about the tabpanel. I was imagining the relationship would be that the `tabpanel` only knows about the _value_ of the `tablist`, not necessarily any of the specifics tabs